atd-etl/cris_import/cris_import.py

```diff
@@ -25,8 +25,9 @@ def main(cli_args):
 
     logger.info(f"{len(extracts_todo)} extract(s) to process")
 
-    if not extracts_todo:
-        return
+    if cli_args.s3_download and not extracts_todo:
+        # always short circuit if we find nothing in S3
+        raise Exception("No extracts found in S3 bucket")
```

Comment:
Eeek—I thought we already had a check like this in place but it must have dropped off during one of the later refactors

Reply:
I also could have sworn we were raising an exception already